floskell.toml

```diff
@@ -0,0 +1,334 @@
+    where         = true
+
+    # The following option controls the limits of tabstop alignment.
+    # According to Floskells documentation the numbers are to be interpreted
+    # as follows.
+    #
+    # > the first integer is a number of characters (absolute limit) and the
+    # > second value is a percentage (relvative limit).
+    # > Alignment will be disabled if the length of any inserted padding is
+    # > larger than absolute limit and is larger than relative limit of the
+    # > maximum field length (tabstop column minus indentation).
+    #
+    # We are using the default configuration, i.e. an absolute limit of 10
+    # characters and a relative limit of 25%. We may want to tweak these
+    # values in the future, though.
+    limits = [10, 25]
+
+  # The following rules control where to insert whitespace in parentheses,
+  # backets and braces.
+  [formatting.group]
+    # By default there are spaces after the opening delimiter and before the
+    # closing delimiter of a group. This rule applies to record constructors
+    # for example.
+    [formatting.group.default]
+      force-linebreak = false
+      spaces          = "both"
+      linebreaks      = "after"
+
+    # No whitespace is added to parentheses in expressions, types and contexts.
+    [formatting.group."("]
+      force-linebreak = false
+      spaces          = "none"
+      linebreaks      = "after"
+
+    # However, whitespace is added to parentheses in import and export
+    # specifier lists.
+    [formatting.group."( in other"]
+      force-linebreak = false
+      spaces          = "both"
+      linebreaks      = "after"
+
+    # There are no spaces after and before the delimiters of list literals.
+    [formatting.group."["]
+      force-linebreak = false
+      spaces          = "none"
+      linebreaks      = "after"
+
+    # For some reason, the rule above does not apply to list types. Thus, it
+    # has to be replicated explicitly.
+    [formatting.group."[ in type"]
+      force-linebreak = false
+      spaces          = "none"
+      linebreaks      = "after"
+
+  # The following rules control where to insert whitespace in around operators.
+  [formatting.op]
+    # By default there is a space before and after an infix operator. If an
+    # infix application has to be broken over multiple lines, the operator
+    # is moved to the next line.
+    [formatting.op."default"]
+      force-linebreak = false
+      spaces          = "both"
+      linebreaks      = "before"
+
+    # There is no space before the comma in tuple types and literals.
+    [formatting.op.","]
+      force-linebreak = false
+      spaces          = "after"
+      linebreaks      = "before"
+
+    # In lambda expressions and `case` expression alternatives, the arrow is
+    # not broken into the next line.
+    [formatting.op."-> in expression"]
+      force-linebreak = false
+      spaces          = "both"
+      linebreaks      = "after"
+
+  # The following options.
```

Comment:
```suggestion
  # The formatting options.
```

Reply:
Oops, I think I forgot to complete the sentence :joy: